cartoframes/dataset_info.py

```diff
@@ -0,0 +1,66 @@
+import time
+
+from carto.datasets import DatasetManager
+from carto.exceptions import CartoException
+
+
+class DatasetInfo():
+    PRIVATE = 'PRIVATE'
+    PUBLIC = 'PUBLIC'
+    LINK = 'LINK'
+
+    def __init__(self, carto_context, table_name):
+        self._get_metadata(carto_context, table_name)
+
+        if self._metadata is not None:
+            self.privacy = self._metadata.privacy
+            self.name = self._metadata.name
+        else:
+            raise CartoException('Something goes wrong accessing the table metadata.')
```

Comment:
Same comment as above:

we could move the raise at the beginning and avoid the `if ... else`

Also what about an error message like:

`We couldn't get the table metadata. Please, try again in a few seconds or contact support for help`

Reply:
:+1: